tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -64,7 +68,7 @@ def create_from_wav_file(cls,
     # `from pybind11_abseil import status`
     # see https://github.com/pybind/pybind11_abseil#abslstatusor.
     audio = _LoadAudioBufferFromFile(
-      file_name, buffer_size, np.zeros([buffer_size]))
+      file_name, sample_count, np.zeros([sample_count]))
```

Comment:
Can you test what happens when loading a WAV file with 2 channels or more? It looks like the array is resized in the C++ so I wonder if we need to reshape the numpy array after loading the WAV file to account for the channel dimension.

https://github.com/tensorflow/tflite-support/blob/e109aa0094e6fd5bb1e30c61ec9b19194c452ac3/tensorflow_lite_support/cc/task/audio/utils/wav_io.cc#L208

Reply:
@khanhlvg I tried it out on a 16-bit stereo WAV file like [this](https://freewavesamples.com/files/Ensoniq-ZR-76-01-Dope-77.wav) one here just now and it works fine.

Reply:
https://freewavesamples.com/ensoniq-zr-76-01-dope-77-bpm

Reply:
@khanhlvg The returned shape for `tensor.buffer` NumPy array turns out to be this.
![image](https://user-images.githubusercontent.com/42828719/159884902-64a02f54-b2e9-47c7-b440-9a8f72de3535.png)

Reply:
Interesting. What happens if you pass in a np.zeros([1]) to the _LoadAudioBufferFromFile function? If it still works, then let use np.zeros([1]) and add a comment that the numpy array will be resized to make it clear for other maintainers of the library.

Then as a sanity test, could you play back the numpy array (e.g. using sounddevice.play) and check if the audio sounds correct?

Reply:
@khanhlvg It doesn't seem to work with `np.zeros([1])`. The test crashes. On the other hand, I'll check if the audio comes out fine when loaded.

Reply:
@khanhlvg I just tested how it sounds using the following snippet and I can hear the same beat on my end.
```python
import sounddevice as sd
sd.play(tensor.buffer)
time.sleep(10)
```